### ISSUE TITLE
Testing file for example tuftool commands

### ISSUE THREAD

**Issue by ecpullen** [awslabs/tough#212]
We have documentation with examples for using tuftool, but no guarantee that the examples continue to stay updated. We should add a test file containing the example commands so that the documentation remains updated.

**Comment by tjkirch** [comment]
If we use something like `cargo-readme` to generate docs from source code, we can use Rust's doc tests to automate this without duplication.

**Comment by webern** [comment]
Yeah, that's what I was thinking (I think I wrote the comment that prompted this issue). Not sure if Cargo Readme can be used to collate separate sources into one readme.  But if not, maybe get all of the documentation into lib.rs.
